src/TicketService.java

```diff
@@ -1,16 +1,33 @@
 import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class TicketService {
-    public static void main(String[] args) {
-        Ticket emptyTicket = new Ticket();
-        System.out.println("Empty Ticket: " + emptyTicket);
+    private Map<String, Ticket> tickets;
+
+    public TicketService() {
+        tickets = new HashMap<>();
+        for (int i = 1; i <= 10; i++) {
+            String id = String.format("%04d", i);
+            Ticket ticket = new Ticket(id, "Hall" + (i % 3 + 1), 100 + i, System.currentTimeMillis(), false, (char)('A' + i % 3), 5.0, new TicketPrice(100, 10));
+            tickets.put(id, ticket);
+        }
+    }
 
-        TicketPrice fullTicketPrice = new TicketPrice(150, 0);
-        Ticket fullTicket = new Ticket("AB12", "BigHall", 123, Instant.now().getEpochSecond(), true, 'B', 5.5, fullTicketPrice);
-        System.out.println("Full Ticket: " + fullTicket);
+    public Ticket getTicketById(String id) {
+        return tickets.get(id);
+    }
 
-        Ticket limitedTicket = new Ticket("SmallHall", 456, Instant.now().getEpochSecond());
-        System.out.println("Limited Ticket: " + limitedTicket);
-        //test
+    public List<Ticket> getTicketsByStadiumSector(char stadiumSector) {
+        return tickets.values().stream()
+                .filter(ticket -> ticket.getStadiumSector() == stadiumSector)
+                .collect(Collectors.toList());
+    }
+
+    public static void main(String[] args) {
+        TicketService ticketService = new TicketService();
+        System.out.println("Tickets in sector A: " + ticketService.getTicketsByStadiumSector('A'));
```

Comment:
For now it is ok but in general using System.out.println for logging in production code is not recommended